x/ibc/core/keeper/msg_server.go

```diff
@@ -127,16 +127,15 @@ func (k Keeper) SubmitMisbehaviour(goCtx context.Context, msg *clienttypes.MsgSu
 func (k Keeper) ConnectionOpenInit(goCtx context.Context, msg *connectiontypes.MsgConnectionOpenInit) (*connectiontypes.MsgConnectionOpenInitResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	if err := k.ConnectionKeeper.ConnOpenInit(
-		ctx, msg.ConnectionId, msg.ClientId, msg.Counterparty, msg.Version,
-	); err != nil {
+	connectionID, err := k.ConnectionKeeper.ConnOpenInit(ctx, msg.ClientId, msg.Counterparty, msg.Version)
```

Comment:
I am thinking of adding this to MsgConnectionOpenInitResponse but I need to check if it'd be useful for the relayer first